packages/api-core/src/site-activity-log/fetchers.ts

```diff
@@ -18,16 +18,36 @@ export async function fetchSiteActivityLog(
 
 export async function fetchSiteRewindableActivityLog(
 	siteId: number,
-	{ number, aggregate = false }: { number: number; aggregate?: boolean }
+	{
+		number,
+		aggregate = false,
+		after,
+		before,
+	}: {
+		number: number;
+		aggregate?: boolean;
+		after?: string;
+		before?: string;
+	}
 ): Promise< ActivityLog > {
+	const params: Record< string, string | number | boolean > = {
+		number,
+		aggregate,
+	};
+
+	if ( after ) {
+		params.after = after;
+	}
+
+	if ( before ) {
+		params.before = before;
+	}
+
 	return wpcom.req.get(
 		{
 			path: `/sites/${ siteId }/activity/rewindable`,
 			apiNamespace: 'wpcom/v2',
 		},
-		{
-			number,
-			aggregate,
```

Comment:
The undefined properties will be filtered out by default so we can just add `before` and `after` here directly.

Reply:
Indeed, thanks for the suggestion. In the `Overview` page I'm seeing it being called with `{number: 1, aggregate: false, after: undefined, before: undefined}` and the query is just `/wpcom/v2/sites/:id/activity/rewindable?_envelope=1&number=1&aggregate=false`, without `before`/`after` parameters.